wurzel/steps/qdrant/step.py

```diff
@@ -240,12 +325,16 @@ def __construct_next_collection_name(self) -> str:
         return f"{self.settings.COLLECTION}_v{previous_version + 1}"
 
     def _get_collection_versions(self) -> dict[int, str]:
+        version_pattern = re.compile(rf"^{self.settings.COLLECTION}_v(\d+)$")
```

Comment:
Add a comment explaining the regex, if possible with examples (what matches and what not)